lib/ip_wrangler/main.rb

```diff
@@ -1,26 +1,22 @@
-$config = YAML.load_file('config.yml')
+$config = YAML.load_file('./config.yml')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.